src/applications/check-in/locales/en/translation.json

```diff
@@ -303,5 +303,19 @@
   "or-you-can-still-file-your-claim": "Or you can still file your claim within 30 days through the Beneficiary Travel Self-Service System (BTSSS).",
   "claim-submitted_one": "Claim submitted",
   "claim-submitted_other": "Claims submitted",
+  "we-cant-file-a-claim": "We can’t file a travel reimbursement claim for you in this tool right now. But you can still file within 30 days of the appointment.",
+  "we-cant-match-your-information": "We can’t match your information to our records",
+  "were-sorry-we-couldnt-match-your-information": "We’re sorry. We couldn’t match your information to our records.",
+  "you-can-still-file-within": "You can still file within 30 days of the appointment through the Beneficiary Travel Self Service System (BTSSS).",
+  "were-sorry-we-couldnt-find-an-account": "We’re sorry. We couldn't find an account that matches that last name or date of birth. Please try again.",
```

Comment:
this seems the same as `sorry-we-couldnt-find-an-account-that-matches-last-name-or-dob` value